poi/src/test/java/org/apache/poi/poifs/dev/TestPOIFSDump.java

```diff
@@ -170,7 +170,7 @@ public void checkPermission(Permission perm) {
     @SuppressForbidden("tests java.security features deprecated in java 17 - no other option though")
     void testMainNoArgs() {
         Assumptions.assumeFalse(POITestCase.getJDKVersion() >= 18,
-                "SecurityManager does not work any more since JDK 18");
+                "SecurityManager does not work since JDK 18");
```

Comment:
in principle, this should be `anymore`, but in practice, it's superfluous, so I removed it.